Aptoide/appcoins-wallet-android
# [MOB 894] Gamification Incentives

**What does this PR do?**

User's gamification progress is shown during the payment flow when making an in app purchase.
Depending on whether the purchase is enough to achieve the next level or not, the description string will change.
Each level has a percentage to when the gamification should be shown or not.

**Database changed?**

   No

**Where should the reviewer start?**

PaymentMethodsFragment.kt

**How should this be manually tested?**

Perforn im app purchases and check if the gamification is shown or not for: Credit Card; Ask someone to pay; PayPal.
Accordingly to each level, check if the gamification is shown depending on the percentage the user has spent in that level.
Check if it shows the right amount of APPC's left to achieve the next level.
Check if the purchase is enough to achieve the next level then the string will change to "Level up with this purchase"

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/MOB-894

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
